bossbot/karma.py

```diff
@@ -1,23 +1,30 @@
+"""
+TODO:
+* Track increments and decrements separately
+* Print the total karma and breakdown
+* Handle emoji reactions to messages :++: and :--:
+"""
+
 from bossbot.bot import BossBot
 from bossbot.db import KarmaDB
 
+def get_target_user(message):
+    if len(message.mentions) < 2:
+        raise ValueError('No user mentioned in karma command.')
+    return message.mentions[1]
```

Comment:
how about we give the user a nice message to tell them what they did wrong? an 💢 emoji is just to let us developers know an unhandled error happened. it's not useful feedback to users